Utilities/macrecovery/macrecovery.py

```diff
@@ -453,6 +455,29 @@ def action_guess(args):
     print(f'UNKNOWN: Failed to determine supported models for MLB {mlb}!')
     return None
 
+def get_basesystem_version(args):
+    print('Extracting recovery version...')
+    os.system(f'hdiutil mount -plist {args.outdir}/BaseSystem.dmg > mount.plist')
```

Comment:
This breaks compatibility for Windows and Linux.

Reply:
O_O I completely forgot about it. Should I use a conditional like `if platform.system() == 'Darwin'` and execute the whole code, or whatever?